src/libproc_macro/lib.rs

```diff
@@ -93,13 +93,13 @@ impl !Sync for LexError {}
 
 impl TokenStream {
     /// Returns an empty `TokenStream` containing no token trees.
-    #[unstable(feature = "proc_macro", issue = "38356")]
+    #[stable(feature = "proc_macro_lib2", since = "1.29.0")]
```

Comment:
Are you going to use `proc_macro_lib3/4/5/...` for the next portions of stabilized APIs? :)
I used `proc_macro_12` in my PR (https://github.com/rust-lang/rust/pull/50473#issuecomment-389332552), because it's Macros 1.2.
(OK, it doesn't matter anyway.)